lib/index.ts

```diff
@@ -55,6 +55,13 @@ export interface DeboredOptions {
   readonly resources?: ResourceRequirements;
 }
 
+// TODO: Need different approach if we support multiple ingress controllers in the future
```

Comment:
why? could be just another entry in the enum, no?

Reply:
Maybe I should add this comment here: https://github.com/toricls/cdk8s-debore/pull/3/files#diff-13b5b151431c7e7a17f273559ed212d5R224

So let's say if we support AWS ALB Ingress controller in the future, the annotations for the Ingress resource will be completely different.